base/docs/helpdb/Base.jl

```diff
@@ -8908,19 +8856,17 @@ vecnorm
 """
     isvalid(value) -> Bool
 
-Returns `true` if the given value is valid for its type, which currently can be one of
-`Char`, `String`, `UTF16String`, or `UTF32String`.
+Returns `true` if the given value is valid for its type, which currently can be either
+`Char` or `String`.
```

Comment:
@StefanKarpinski, `Base` still includes UTF-8 validation: `isvalid(String, "foo")` works, and calls the C function `u8_isvalid`.

You just removed `isvalid(::String)`.  This seems a bit arbitrary and probably a mistake?  Maybe we should have a fallback method `isvalid{T}(x::T) = isvalid(T, x)` ?